build.sbt

```diff
@@ -10,7 +10,7 @@ version := "0.4.1-SNAPSHOT"
 
 scalaVersion := "2.10.4"
 
-sparkVersion := sys.props.get("spark.version").getOrElse("1.4.1")
+sparkVersion := sys.props.get("spark.version").getOrElse("1.5.0-SNAPSHOT")
```

Comment:
Why change the default Spark version to 1.5 unless we're dropping support for 1.4.x?